CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [[v10.?.?](https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v10.0.0...HEAD)] - 2020-xx-xx
 
-- Write your awesome change here (GH-xxxx)
+- Generate `aws-auth` ConfigMap's roles from Object. No more string concat. (by @dpiddockcmp)
```

Comment:
Please remove your modification from the Changelog.

Reply:
Done